codalab/lib/codalab_manager.py

```diff
@@ -223,6 +223,11 @@ def init_config(self, dry_run=False):
             config['server']['engine_url'] = "sqlite:///{}".format(sqlite_db_path)
             print "Using SQLite database at: {}".format(sqlite_db_path)
 
+        # Generate secret key
+        config['server']['secret_key'] = get_random_string(
+            48, "=+/abcdefghijklmnopqrstuvwxyz"
+                "ABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789")
```

Comment:
How should we help people generate this if they already have a config file from a previous installation?